lib/src/analyzers/lint_analyzer/rules/flutter_rule_utils.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:analyzer/dart/element/type.dart';
```

Comment:
@incendial do we plan covers this functions by tests?

Reply:
Since it's now covered by multiple rule tests, coveting it won't bring any value.